include/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM php:8.1.0-zts-bullseye AS php-8-1-x64-zts-linux-build
+RUN apt update
+RUN apt install libffi-dev
+RUN docker-php-ext-install ffi
+RUN curl http://131.123.42.38/lmcrs/v1.0.0/srcml_1.0.0-1_ubuntu20.04.deb > srcml.deb
+RUN apt install -y ./srcml.deb
+COPY process.php .
+COPY symbols.txt .
+RUN php process.php linux
+FROM scratch AS php-8-1-x64-zts-linux
+COPY --from=php-8-1-x64-zts-linux-build /engine-8-1-x64-zts-linux.h /
+COPY --from=php-8-1-x64-zts-linux-build /constants-8-1-x64-zts-linux.php /
+
+FROM php:8.1.0-bullseye AS php-8-1-x64-nts-linux-build
+RUN apt update
+RUN apt install libffi-dev
+RUN docker-php-ext-install ffi
+RUN curl http://131.123.42.38/lmcrs/v1.0.0/srcml_1.0.0-1_ubuntu20.04.deb > srcml.deb
```

Comment:
Unfortunately the official srcML download link is HTTP - https://www.srcml.org/ - not an issue at runtime unless you're regenerating the headers but it's definitely a security issue.